src/main/java/com/bitso/Bitso.java

```diff
@@ -161,16 +161,25 @@ public BitsoFee getUserFees(){
         return new BitsoFee(o);
     }
 
-    public BitsoLedger getUserLedger(String... specificOperation){
+    public BitsoLedger getUserLedger(){
         String request = "/api/v3/ledger";
-        if(specificOperation.length == 1){
-            request += "/" + specificOperation[0];
+        log(request);
+        String json =  sendBitsoGet(request);
+        JSONObject o = Helpers.parseJson(json);
+        if(o == null || o.has("error")){
+            logError("Error getting user ledgers");
+            return null;
         }
+        return new BitsoLedger(o);
+    }
```

Comment:
Creo que siempre es mejor pasar `null` (and handle that way) en vez de mandar empty strings, no?